packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -77,15 +84,15 @@ export interface ProjectMetaState {
   hasCypressEnvFile: boolean
   hasValidConfigFile: boolean
   hasSpecifiedConfigViaCLI: false | string
-  hasMultipleConfigPaths: boolean
+  allFoundConfigFiles: string[]
```

Comment:
```suggestion
  allFoundConfigFiles: readonly string[]
```

Prevents `push` (mutate) but allows reassign.

Reply:
Not doing this because we want to push into it.